definitions/service.md

```diff
@@ -1,17 +1,27 @@
 ---
 title: Service
-status: Feedback Appreciated
-category: concept
+status: Completed
+category: Concept
 ---
 ## Service
 
+### What it is
+
 A “Service” is a term with more meanings than virtually any other used in IT today. Traditionally a service represented a software system of which requests were made, as opposed to ‘fat client’ applications where all logic was contained in the program running on a user’s desktop. 
```

Comment:
Thanks for your contribution! I tweaked your version a little and provided feedback but still need to review with Jason on Friday, so please do not take action until I had the chance to do so.

**Suggested update:**
Service can have multiple meanings. Traditionally, service refers to software processes whose "services" can be requested remotely. Take Gmail, for example. You access it through your browser or mobile app and, to send and receive messages, the web application sends and receives requests to/from remote services globally distributed on Google's servers. This approach differs from older 'fat client' apps where all logic is contained in the program running on the user’s computer. 

Please note, in the context of Kubernetes, service has a different meaning. You can refer to it in the [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/
). 

Preliminary feedback:

- I tweaked the language a little and added an example
- The Kubernetes definition is very specific and too technical for this glossary. It's a good point to mention that it does have a different meaning, though. But for our purposes, it's enough to point to the k8s docs.
- I didn't include the web service and database mention since (as far as I understand it) they are merely examples of a traditional "service" >> remote services that can be accessed from a user's computer (am I right or missing something, Jason?) Does it have additional meanings we should include?

Questions for @JasonMorgan:

- Does 'whose "services" can be requested remotely' sound weird? I was trying to highlight why it's called like that. Maybe there's a better way to phrase it
- See question regarding web apps and databases above